web_console_v2/api/requirements.txt

```diff
@@ -18,7 +18,7 @@ elasticsearch==7.11.0
 flatten-dict==0.3.0
 pymysql==1.0.2
 setuptools==41.0.0
-tensorflow==2.4.0
+tensorflow==2.4.1
```

Comment:
Could you please elaborate more about the behaviors? What kinds of errors you are encountering?